Sikorsky Alexandr/1/lib/runner.rb

```diff
@@ -0,0 +1,49 @@
+require 'yaml'
+require_relative 'wordplay_bot'
+require_relative 'db_wrapper'
+require_relative 'parser.rb'
+
+class Runner
+  BASE_PAGES = ['https://www.bydewey.com/pun.html', 'https://www.bydewey.com/pun2.html'].freeze
+  CONFIG_FILE_PATH = 'config.yaml'.freeze
+
+  def run
+    database = DBWrapper.new
+
+    return if check_connection(database).nil?
+
+    fill_database(database) if database.empty?
+
+    token = load_token
+
+    raise 'empty token' if token.nil?
+
+    bot = WordplayBot.new(database, token)
+
+    bot.start
+  end
+
+  private
+
+  def fill_database(database)
+    parser = Parser.new
+    wordplays = BASE_PAGES.reduce([]) { |stach, url| stach << parser.parse(url) }
```

Comment:
avoid using reduce, it makes code harder to read
https://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-flat_map